fetch.bs

```diff
@@ -9062,6 +9062,107 @@ done only by navigations). The <a>fetch controller</a> is also used to
 <a for=request>redirect mode</a> set to "<code>manual</code>".
 
 
+<h3 id=fetch-elsewhere-security>Same-origin policy when using fetch</h3>
+
+<p>The Fetch standard enforces some of the mechanisms that constitute the Same Origin Policy (SOP),
+which is a critical aspect of web security. Specifically, Fetch imposes SOP restrictions on network
+access.
+
+<p>In a nutshell, when a user accesses a certain origin using their browser or other user agent,
+they don't expect that the user agent implicitly grants that <a for=/>origin</a> full access to
+their network, which could, for example, be an intranet, despite the user agent itself having that
+access.
+
+<p>In addition, the user agent may include footprint that identifies the user to the origin,
+explicitly using cookies or client certificates, or implicitly using, for example, the user-agent's
+IP address.
```

Comment:
How is this connected to the same-origin policy? A site that wants to leak this kind of information can always embed an `<iframe>`.

Reply:
Imagine any domain had read access to `corp-intranet.com/about-my-computer.json` and it gave you all kinds of information about yourself, identifying you by your IP address.